helpers.sh

```diff
@@ -7,8 +7,12 @@ function status { echo == $(date "+%F %T") $* ; }
 
 function run { set -o pipefail; status "Running: $*"; $* 2>&1 | indent ; }
 
+function dns_base {
+	echo $H_RUN_URL | cut -d/ -f3 | sed -e 's/api.//'
+}
+
 function docker_login {
-	hub_fio=$(echo $H_RUN_URL | cut -d/ -f3 | sed -e 's/api./hub./')
+	hub_fio="hub.$(dns_base)"
```

Comment:
Why not use `FIO_HUB_URL` here?

Reply:
it would be the other way around, but FIO_HUB_URL is set in bb-config.sh. Its only available to a yocto build. this is getting setting for container builds.

Reply:
I understand it. My point is that defining the env. var. at the beginning of the overall script (maybe publish.sh) is less subtle then defining `hub_fio` in one of the functions and then using it at the other places.

Reply:
we've never defined variable globally in something like helpers.sh so that multiple scripts get a value. We could do something like the `dns_base` function in helpers.sh though. Then everyone has access to it.

Reply:
I've cleaned this up a little bit and just set dns_base and hub_fio as environment variables. gonna test this stuff out and then look at the unit-test